### DISCUSSION TITLE
Wally and Joe setting Curation Priorities

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/RiverscapesXML#142] [General] [answered]
We have a general goal of "Curating" our most important project types from @Riverscapes/etal-fhc by September 1. We know that curation means:
- Completing symbology for QRAVE, ARcRAVE and WebRAVE 
- Curating Project Views
- Curating business logic
- Completing and curating useful HTML reports.

## What Projects

@Riverscapes/etal-fhc shall be responsible for:
- pyBRAT
- sqlBRAT
- pyRCAT
- sqlRCAT
- Riverscapes Context
- RIM

@shelbysawyer will do VBET & HAND
@lauren-herbine will do Confinement & RiverStyles

@nick4rivers and @chafen will do:
- LTPBR Design
- LTPBR Planning
- RS Assessment
- and TBD

## To define / do
- [ ] @wally-mac and @joewheaton should add to Riverscapes Org all the paying USU projects (A index, sponsor, name). Therein we should start triage (to-do, in-progress)
- [ ] @wally-mac and @joewheaton should add milestones tied to big release goal and project completions, as well as weekly milestones for pacing. 
- [ ] For @Riverscapes/etal-fhc assignments @wally-mac should use the new symbology curation template by layer or logical group of layers and assign and/or task out as appropriate. 
- [x] For Anabranch paying things, @joewheaton will do same and for @lauren-herbine and @shelbysawyer

**Comment by joewheaton** [comment]
@wally-mac do you have time today or tomorrow to revisit this together?

**Reply by wally-mac** [reply]
@joewheaton what does the rest of the week look like for you to revisit this together?  I'm around all week and I'm flexible.

**Comment by joewheaton** [comment]
@wally-mac, just a reminder of above tasks...

**Comment by philipbaileynar** [comment, marked as answer]
@wally-mac make sure that the techs coordinate with @MattReimer EARLY! Get one legacy BRAT project done and then work with @MattReimer to confirm that it validates and goes into the warehouse smoothly. Then scale up from there. 

Don't whatever you do, have the techs update 60 projects only to find there was a fatal floor and every one of them needs to be tweaked again.

**Reply by wally-mac** [reply]
@Cashe93 please follow @philipbaileynar directive in the past post.

**Reply by Cashe93** [reply]
Sounds good!
